config/api-extractor.json

```diff
@@ -0,0 +1,364 @@
+/**
```

Comment:
The comments in this file are pretty over the top. But also they are pretty helpful for all that information to be located directly inside the config... as long as it doesn't change